product_docs/docs/pgd/5/raft/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+
+title: Proxies, Raft and Raft subgroups
+
+---
+
+PGD manages its metadata using a  Raft model where a top-level group spans all the data nodes in the PGD cluster. A Raft leader is elected by the top-level group and propagates the state of the top-level group to all the other nodes in the group. 
+
+!!! Hint What is Raft?
+Raft is an industry accepted algorithm for making decisions though achieving “consensus” from a group of separate nodes within a distributed system. 
+!!!
+
+It is essential for certain operations in the cluster that the Raft leader – such as adding and removing nodes and allocating ranges for [galloc](sequences/#pdg-global-sequences) sequences – is both established and maintained. 
+
+It also means that one half of the nodes in the top-level group, plus one additional node must be able to reach each other. So, in a top-level group with five nodes, at least three of the nodes must be reachable by each other to establish a Raft leader.
+
+## Raft and proxy routing
+
+One function that also uses Raft is proxy routing. Proxy routing requires that the proxies are able to coordinate writing to a data node within their group of nodes. This data node is the write leader. If the write leader goes offline, the proxies need to be able to switch to a new write leader, selected by the data nodes, to maintain continuity for connected applications.
+
+Proxy routing can be configured on a per node group basis in PGD 5, but the recommended configurations are "global" and "local" routing.
+
+Global routing uses the top-level group to manage the proxy routing. All nodes in the group are eligible to become write leader for all proxies. Connections to proxies within the top-level group will be routed to data nodes within the top-level group.
+
+Local routing uses the subgroups, often mapped to locations, to manage the proxy routing within the subgroup. This allows subgroups to have their own Raft leader for proxy routing and, in turn, own write leader. Connections to proxies within the subgroup will then route to data nodes within the subgroup. 
+
+## Raft subgroups
+
+In previous versions of PGD, HARP manager elected the write-leader and relied on the top-level Raft which powers decision making between nodes in the group. But issues could occur if connectivity between the subgroups was completely cut off. With a total number of reachable nodes reduced to half of what it was, there is no way that the Raft consensus can become quorate, and without that, it would be unable to make decisions and elect leaders. 
```

Comment:
Rereading this again, even after the changes, I kind of wonder whether we should be talking about past (if somebody cares they can read past docs), and use this "we did this badly before, so we do it this way now", maybe it's better to just say "we do things this way and it brings us these and these benefits" (that's general problem of our doc in some places)

Reply:
Basically, replace this and next para with a much better worded version of:

PGD allows reads and asynchronous DMLs to work even when global consensus is lost, and the local routing is commonly used for geographical separation (or sharding) of writes, it's important to continue routing even in the lack of  global consensus.

And then we can continue with "To address this, PGD 5"

Reply:
Re-organized - will request a re-review